cosmosdb/types.bal

```diff
@@ -340,11 +344,12 @@ public type StoredProcedureExecuteOptions record {|
     int|float|decimal|string partitionKey?;
 |};
 
-# Represent the optional parameters which can be passed to the function when reading the information about other 
+# Optional parameters which can be passed to the function when reading the information about other 
 # resources in Cosmos DB such as Containers, StoredProcedures, Triggers, User Defined Functions, etc.
 # 
-# + consistancyLevel - The consistency level override. Allowed values are `Strong`, `Bounded`, `Session` or `Eventual`.
-#                      The override must be the same or weaker than the account’s configured consistency level.
+# + consistancyLevel - The consistency level override
```

Comment:
Suggestion: "The consistency level override. Possible values are `Strong`, `Bounded`, `Session`, and `Eventual`. The override must be the same or weaker than the Cosmos DB account’s configured consistency level."